msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -2291,5 +2394,33 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
         val result = application.signUp(invalidUsername)
         assertTrue(result is SignUpError)
         assertTrue((result as SignUpError).isInvalidUsername())
+
+        checkSafeLogging()
+    }
+
+    private fun checkSafeLogging() {
+        val piiTrueToCheck = listOf(
+            "password:","password=",
```

Comment:
let's make this list longer.
- `"password"` (so without : and =)
- `'"password"'` (so looking for the password value that includes " in the string, for example when JSON is outputted it will contain ")
- add snake_case as well as camelCase for multi-word values like continuation token.

Reply:
Have updated the rules and scopes.

Reply:
Should (user) attributes be added?